.github/workflows/ci.yml

```diff
@@ -87,6 +87,7 @@ jobs:
       uses: actions/setup-python@v4
       with:
         python-version: ${{ matrix.pyver }}
+        allow-prereleases: true
```

Comment:
```suggestion
```
3.13 has been released already

Reply:
Please don't revert that, we then have to remember to add it back in every time we add a new version. The option only uses prereleases when there are no stable releases, so removing it achieves nothing.